src/app/static/src/tests/projects/actions.test.ts

```diff
@@ -436,9 +436,30 @@ describe("Projects actions", () => {
         actions.promoteVersion({commit, state, rootState, dispatch} as any, versionPayload);
 
         setTimeout(() => {
-
             const posted = mockAxios.history.post[0].data;
             expect(posted).toEqual("name=newProject");
+
+            expect(dispatch.mock.calls[0][0]).toBe("getProjects");
+            expect(commit.mock.calls.length).toBe(1);
```

Comment:
There shouldn't be any commit calls for the success state as we're ignoring success. So this should be checking for length 0. I think this case is actually going through the error route because of the post body matching failing - I tried fixing the mockAxios.onPost locallt to check for data name=newProject as discussed, and that gives a commit call count of 0.